rotkehlchen/tests/unit/decoders/test_curve.py

```diff
@@ -517,7 +520,7 @@ def test_curve_remove_liquidity_with_internal(database, ethereum_transaction_dec
     with database.user_write() as cursor:
         dbevmtx.add_evm_transactions(cursor, [transaction], relevant_address=None)
         dbevmtx.add_evm_internal_transactions(cursor, [internal_tx], relevant_address=location_label)  # noqa: E501
-    events = ethereum_transaction_decoder.decode_transaction(
+    events, _ = ethereum_transaction_decoder.decode_transaction(
```

Comment:
is there any test where the value for the refresh is true?

Reply:
Check below